x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DatafeedJobsRestIT.java

```diff
@@ -113,43 +113,30 @@ private void addAirlineData() throws IOException {
         StringBuilder bulk = new StringBuilder();
 
         Request createEmptyAirlineDataRequest = new Request("PUT", "/airline-data-empty");
+        // space in 'time stamp' is intentional
         createEmptyAirlineDataRequest.setJsonEntity(
-            "{"
-                + "  \"mappings\": {"
-                + "    \"properties\": {"
-                + "      \"time stamp\": { \"type\":\"date\"}," // space in 'time stamp' is intentional
-                + "      \"airline\": { \"type\":\"keyword\"},"
-                + "      \"responsetime\": { \"type\":\"float\"}"
-                + "    }"
-                + "  }"
-                + "}"
+            """
```

Comment:
Would it be possible to preserve indentation? Just like you did in line 129?

BTW, do you do this transformation using some automated tool or by hand?